test/prototype_transforms_kernel_infos.py

```diff
@@ -384,7 +396,10 @@ def reference_inputs_resize_bounding_box():
             reference_fn=reference_resize_bounding_box,
             reference_inputs_fn=reference_inputs_resize_bounding_box,
             closeness_kwargs={
-                (("TestKernels", "test_against_reference"), torch.int64, "cpu"): dict(atol=1, rtol=0),
+                # FIXME: those are crazy tolerances. We need to investigate what is going on
```

Comment:
Per comment

Reply:
I'm surprised we have to change the `resize` tests. Resize shoudn't put (part of ) a bbox outside the image?

Reply:
:facepalm: I misread what @vfdev-5 commented offline. This was one of the few exceptions. Let me revert here. Thanks for the heads-up.

Reply:
but then, how come we ended up having differences here? Even if we clamp after a resize, the bbox should still be the same?

Reply:
Good point. I'll look into that.